js/private/test/BUILD.bazel

```diff
@@ -35,8 +35,8 @@ write_source_files(
         "shellcheck_launcher.sh": ":shell_launcher_sed",
     },
     target_compatible_with = select({
-        "@aspect_bazel_lib//lib:bzlmod": ["@platforms//:incompatible"],
```

Comment:
Why switch this one? Should generate+checkin both instead?

Reply:
Yup. Better to just add both separate targets and test both. ✅